pom.xml

```diff
@@ -156,7 +156,7 @@
     <mockito.version>1.10.19</mockito.version>
     <mysql.version>5.1.21</mysql.version>
     <netty.http.version>1.7.0</netty.http.version>
-    <netty.version>4.1.75.Final</netty.version>
+    <netty.version>4.1.16.Final</netty.version>
```

Comment:
Why are we downgrading netty?

Reply:
It was updated recently as a part of security vulnerability fix but it has been causing build failures, hence it has been downgraded temporarily. Update of Spark2 which is currently ongoing will fix this but it will most probably be merged later after this PR. This downgrade will be reverted once those changes are merged with develop.